code/clue.js

```diff
@@ -1,59 +1,357 @@
-// STEP 1 - CREATE OBJECTS FOR ALL THE SUSPECTS, SOMETHING LIKE THIS:
+let killerIsPicked = false;
+let weaponIsPicked = false;
+let roomIsPicked = false;
```

Comment:
Good idea to store this info in variables! Nonetheless, one thing could be improved :
There is a strict convention about boolean variables names. They should always start with `is` as a prefix: `isKillerPicker`, `isWeaponPicked`, `isRoomPicked`